functions_and_scope_lab.js

```diff
@@ -0,0 +1,262 @@
+// # Function Exercises
+//
+// ## Question 1
+//
+// Write a function that returns the largest of two numbers. If they are equal, return the second one.
+//
+// Example:
+//
+// Input: `12, 4`
+//
+// Output: `12
+const largestNum = (num1,num2) => {
+  if (num1 > num2) {
+    return num1
+  } else {
+    return num2
+  }
+}
+console.log(largestNum(12,4))
+
+// ## Question 2
+//
+// Write a function named getDogAge that takes one argument: a dog's age in human years. The function returns the dog's age in dog years. The conversion rate is 1 human year to 7 dog years.
+//
+// Example:
+//
+// Input: `4`
+//
+// Output: `28`
+const getDogAge = (age) => {
+  return age * 7
+}
+console.log(getDogAge(4))
+
+// ## Question 3
+//
+// Write a function `arraySum` that takes in an array and returns the sum of all the elements.
+//
+// Example:
+//
+// Input: `[13,2,4,-2]`
+//
+// Output: `17`
+const arraySum = (array) => {
+  let result = 0
+  for (a of array) {
+    result += a
+  }
+  return result
+}
+console.log(arraySum([13,2,4,-2]))
+
+// ## Question 4
+//
+// Write a function that takes in any two numbers and return their sum.
+//
+// Example:
+//
+// Input: `14, 5`
+//
+// Output: `19`
+const sum = (num1,num2) => {
+  return num1 + num2
+}
+console.log(sum(14,5))
+
+// ## Question 5
+//
+// Write a function takes in any number grade and returns a corresponding letter grade.
+//
+// | Number Grade | Equivalent Letter Grade |
+// | :--------: | :---------: |
+// | 100 | A+ |
+// | 90 - 99 | A |
+// | 80 - 89 | B |
+// | 70 - 79 | C |
+// | 65 - 69 | D |
+// | Below 65 | F |
+//
+//
+// Example:
+//
+// Input: `88`
+//
+// Output: `"B"`
+const letterGrade = (num) => {
+  if (num === 100) {
+    return 'A+'
+  } else if (num < 100 && num > 89) {
+    return 'A'
+  } else if (num < 90 && num > 79) {
+    return 'B'
+  } else if (num < 80 && num > 69) {
+    return 'C'
+  } else if (num < 70 && num > 64) {
+    return 'D'
+  } else if (num < 65) {
+    return 'F'
+  }
+}
+console.log(letterGrade(65))
+
```

Comment:
Great job! With your knowledge on code control flow. How could you make this code cleaner?